lib/webauth.js

```diff
@@ -277,7 +302,12 @@ module.exports = function(authManager, cookiePort, isSecurePort) {
         initZLUXSession(req);
         req.session.zlux.expirationTime = Date.now() + result.expms;
       }
-      
+      if (result.categories.zss) {
```

Comment:
dont hardcode this.
Just make it so that if any plugin has reason "Expired password" that a 428 is set. You need to go through the list.
Also, you need 428 to be a constant.